### ISSUE TITLE
Profiles without survey responses

### ISSUE THREAD

**Issue by doughardy-ctnj** [mzagaja/ctnewsjunkie_voterguide#83]
We have 40 so far as of about 3 p.m. today

About a third of total respondents failed to fill in survey questions

**Comment by mzagaja** [comment]
Did you want me to change the code to show profiles if there isn't a survey response but they answered a specific profile question? Otherwise not much I can do on my end. For future iterations I think maybe figuring out how to combine the views of those two things so they only need one link to do it could work but that might be a bit tricky on my end.

**Comment by doughardy-ctnj** [comment]
Tough call - that would be a gift on our part but at the same time we want to show participation...

**Comment by doughardy-ctnj** [comment]
I'm not sure that's really fair ... I can email them all to remind them but the problem is, I don't know which link will work for them to add responses. 

I would instruct them to log into their profile using their edit link, and then once logged in, they should click this?
http://vote.ctnewsjunkie.com/surveys/2/survey_responses/new

**Comment by mzagaja** [comment]
Yes that last link should work for them after they log in. So they should log in on vote.ctnewsjunkie.com then click that link.

**Comment by doughardy-ctnj** [comment]
OK thanks - let me email them all before we display their profiles without the questionnaires. Maybe we end up going that way eventually but we'll see.
